aws-codeguruprofiler-profilinggroup/aws-codeguruprofiler-profilinggroup.json

```diff
@@ -80,13 +92,18 @@
     },
     "update": {
       "permissions": [
+        "sns:Publish",
+        "codeguru-profiler:RemoveNotificationChannel",
```

Comment:
The `"codeguru-profiler:AddNotificationChannels"` is also needed here isn't it in case more ARNs are added during the update?